extensions/github/src/search-repositories.tsx

```diff
@@ -16,12 +17,14 @@ function SearchRepositories() {
   const preferences = getPreferenceValues<{ includeForks: boolean }>();
 
   const [searchText, setSearchText] = useState("");
+  const [debouncedSearchText] = useDebounce(searchText, 200);
```

Comment:
how does that improve the throttling of the search? What's the issue in the first place?

Reply:
That one isn't detailed enough indeed. The problem was that with `throttle` on the `List` component, the visited repositories would be filtered asynchronously even though it can be done immediately. Here's a comparison where the 2nd one is slightly faster.

## Before

https://user-images.githubusercontent.com/16003285/202664563-7367fb06-9f08-4384-8422-7622af86e9f2.mp4

## After

https://user-images.githubusercontent.com/16003285/202664285-853a417b-0f1c-41a9-af44-49fb088d56e4.mp4

Reply:
that seems to be because of the `foundRepositories` that you introduced, not the debounce thing

Reply:
Fair enough